frontend/src/Author/Author.ts

```diff
@@ -1,5 +1,7 @@
 import ModelBase from 'App/ModelBase';
 
+export type AuthorStatus = 'continuing' | 'ended' | 'deleted';
```

Comment:
```suggestion
export type AuthorStatus = 'continuing' | 'ended';
```

There isn't a `deleted` yet in Readarr.

Reply:
Also please add `status: AuthorStatus;` before the tags.

Reply:
I removed all the references I added to the `deleted` status, since we don't support that